crates/oxc_diagnostics/src/service.rs

```diff
@@ -128,17 +128,28 @@ impl DiagnosticService {
     /// Wrap [diagnostics] with the source code and path, converting them into [Error]s.
     ///
     /// [diagnostics]: OxcDiagnostic
-    pub fn wrap_diagnostics<P: AsRef<Path>>(
+    pub fn wrap_diagnostics<C: AsRef<Path>, P: AsRef<Path>>(
+        cwd: C,
         path: P,
         source_text: &str,
         source_start: u32,
         diagnostics: Vec<OxcDiagnostic>,
-    ) -> (PathBuf, Vec<Error>) {
-        let path = path.as_ref();
-        let path_display = path.to_string_lossy();
-        // replace windows \ path separator with posix style one
-        // reflects what eslint is outputting
-        let path_display = path_display.cow_replace('\\', "/");
+    ) -> Vec<Error> {
+        #[cfg(test)]
+        let is_jetbrains_terminal = false;
```

Comment:
When running the tests through a JetBrains terminal locally, the snapshots use full file URLs instead of the relative ones. This was my attempt at fixing it, but it doesn't seem to work.

Reply:
we could use debug assertions instead

effectivly
```
        #[cfg(debug_assertions)]
```

Reply:
If anybody has any idea how to ignore the env check during tests, I still need a working solution so that testing within IntelliJ terminals don’t fail the snapshots.